docs/core/testing/unit-testing-published-output.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Test Published Output with dotnet vstest
+description: Learn how to run tests on published output with the dotnet vstest command.
+keywords: MSTest, .NET, .NET Core, xUnit, test, published, output, test after publishing, run tests on dll
+author: kendrahavens
+ms.author: kehavens
+ms.date: 10/18/2017
+ms.topic: article
+ms.prod: .net-core
+ms.devlang: dotnet
+ms.assetid: 3965e4ca-75b8-4969-b3af-ca993c397a15
+---
+
+# Test Published Output with dotnet vstest
```

Comment:
Here, and all headings below should be sentence cased, not title case.

Reply:
Gotcha!